### PR TITLE
add lemmatization

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by dakrone** [comment]
Sorry I haven't gotten to this, I just had a(nother) baby. Is there a reason you closed this?

**Comment by ghost** [comment]
Ok, I thought nobody was interested. It's my first pull request.

File: project.clj [hunk 1]
```diff
@@ -1,6 +1,6 @@
-(defproject clojure-opennlp "0.5.1-SNAPSHOT"
+(defproject org.clojars.dmitrykosykh/clojure-opennlp "0.5.1-SNAPSHOT"
```
Comment (dakrone) [inline project.clj:1]: I think these changes are accidental and should not be included in this PR

File: project.clj [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   :description "Natural Language Processing with Clojure, library for opennlp."
-  :url "http://github.com/dakrone/clojure-opennlp"
+  :url "http://github.com/dmitrykosykh/clojure-opennlp"
```
Comment (dakrone) [inline project.clj:3]: Same here

File: src/opennlp/nlp.clj [hunk 3]
```diff
@@ -25,13 +27,13 @@
   (if (seq span-col)
     (if (string? data)
       (seq
-        (Span/spansToStrings
-          #^"[Lopennlp.tools.util.Span;" (into-array span-col)
-          ^String data))
+       (Span/spansToStrings
```
Comment (dakrone) [inline src/opennlp/nlp.clj:30]: Can you undo these whitespace changes? They aren't needed for this PR

**Review by dakrone — CHANGES_REQUESTED** [review]
I left some comments, thanks for opening this!

File: project.clj (hunk 1 above)
Comment (ghost) [reply project.clj:1]: I can't understand why the second commit was added to PR

File: src/opennlp/nlp.clj (hunk 3 above)
Comment (dmitrykosykh) [reply src/opennlp/nlp.clj]: I'm used cljfmt

**Comment by ghost** [comment]
You don't need to add the second commit I did it for my repo

File: src/opennlp/nlp.clj (hunk 3 above)
Comment (ghost) [reply src/opennlp/nlp.clj:30]: I used cljfmt
